docs/api/system-preferences.md

```diff
@@ -274,3 +282,43 @@ Returns `Boolean` - `true` if an inverted color scheme, such as a high contrast
 theme, is active, `false` otherwise.
 
 [windows-colors]:https://msdn.microsoft.com/en-us/library/windows/desktop/ms724371(v=vs.85).aspx
+
+### `systemPreferences.getEffectiveAppearance()` _macOS_
+
+Returns `String` - Can be `dark`, `light` or `unknown`.
+
+Gets the macOS appearance setting that is currently applied to your application,
+maps to [NSApplication.effectiverAppearance](https://developer.apple.com/documentation/appkit/nsapplication/2967171-effectiveappearance?language=objc)
+
+Please note that because Electron is not built targetting the 10.14 SDK your applications
+`effectiveAppearance` will always default to "light" and never automatically inherit the OS
```

Comment:
Confusing use of 'never' here since we're implying that it will work correctly once Electron is built targeting the 10.14 SDK.

Possibly better:

> Please note that until Electron is built targeting the 10.14 SDK, your application's `effectiveAppearance` will default to 'light' and won't inherit the OS preference. In the interim we have provided a helper method `startAppLevelAppearanceTrackingOS()` which emulates this behavior.

Reply:
Actually -- rather than make this a public API that people will have to opt-in on and which we'll have to deprecate after moving to 10.14 SDK -- why not make this run automatically so that the color scheme will Just Work both before and after the SDK switch?

Reply:
> why not make this run automatically so that the color scheme will Just Work both before and after the SDK switch?

This is a good question, I personally wouldn't want this to be forced upon apps because enabling dark mode in your app is more complex than just the native macOS UI looking dark mode.   For instance if we forced this on all app's context menus would become dark themed, which may not look too good with a light themed app.

In my mind, this would be opt-in for apps that have a dark theme, and once we move to 10.14's SDK the auto-move can be opt-out through a field in the `Info.plist` which we can document once we make that move.  (There is a built-in field macOS respects to not auto-switch that Chrome uses at the moment)

I'll add a note to this API though that it is exempt from the standard deprecation cycle, it will be removed in a single major bump at some point in the future because as soon as we reach 10.14 this API will be pointless and probably not work correctly.